addons/addon-base-raas-ui/packages/base-raas-ui/src/models/aws-accounts/AwsAccountsStore.js

```diff
@@ -13,14 +13,30 @@
  *  permissions and limitations under the License.
  */
 
+import _ from 'lodash';
 import { types } from 'mobx-state-tree';
 import { BaseStore } from '@aws-ee/base-ui/dist/models/BaseStore';
 
-import { getAwsAccounts, addAwsAccount, createAwsAccount } from '../../helpers/api';
+import { getAwsAccounts, addAwsAccount, createAwsAccount, updateAwsAccount } from '../../helpers/api';
 import { AwsAccount } from './AwsAccount';
 import { BudgetStore } from './BudgetStore';
 import Budget from './Budget';
 
+const filterNames = {
+  ALL: 'All',
+  CURRENT: 'Up-to-Date',
+  UPDATEME: 'Needs Update',
+  NEW: 'New',
```

Comment:
It does not, NEW means an account that has been added to SWB but that hasn't been onboarded yet. PENDING is a label I should probably add, I was just thinking about that today.

Reply:
For this PR I'm not sure if it's necessary, since none of the functionality makes use of a PENDING tag, but when the check permissions button is implemented I think a pending tag and functionality would be useful.

Reply:
Got it. Yea, we can further define the status when the functionality is ready. NEW is a bit confusing to be used as a status, maybe we can use something like `ONBOARDME : 'Needs Onboard'`?

Reply:
Yep, that makes sense. I made that change just now